### ISSUE TITLE
logicle transform should only be applied once per compensation matrix

### ISSUE THREAD

**Issue by GoogleCodeExporter** [johnbachman/py-fcm#13]
```
I was talking to Mario Roederer this afternoon, and he explained that the 
logicle transform is only 
applied ONCE per compensation matrix - basically, use the first data set to 
find the cutoff, then 
use the same cutoff for all the other files with the same comp matrix. Can you 
modify fcm so it 
does this for all the data-driven transforms? This will ensure that FCS within 
a set have 
comparable transforms.

Basically, I think you set up a dictionary with the comp-matrix as a key

try:
  r = d[comp_matrix_as_key]
except KeyError:
  r = calc_r
  d[comp_matrix_as_key] = r

```

Original issue reported on code.google.com by `cliburn....@gmail.com` on 4 Feb 2010 at 1:27
